### DISCUSSION TITLE
啥时候发个版呢

### DISCUSSION THREAD

**Discussion by sf467** [waylyrics/waylyrics#242] [General]
太想要提取lrc中歌曲翻译功能的新版了，但是自己编译安装一直在出错，啥时候可以出下一个release呢

**Comment by mokurin000** [comment]
> 太想要提取lrc中歌曲翻译功能的新版了，但是自己编译安装一直在出错，啥时候可以出下一个release呢

v0.3.13 就可以
你是用什么发行版？

**Reply by sf467** [reply]
我等的就是这个，这个话题是我4号发的，
0.3.13是27号出的，没问题
